### PR TITLE
Services/multiregionparticlesdeployment/ma 2253

### PR DESCRIPTION
**Pull request by RajkumarShivage**

### CODE REVIEW

**Comment by chandramishra** [comment]
LG

















LGMT

File: Makefile [hunk 1]
```diff
@@ -1,21 +1,26 @@
+ifdef ACCOUNT
+else
+        ACCOUNT := 442294194136
+        export
+endif
 
-BUILD_SUBDIRS := lib awsconfig cloudtrail awsconfigrules awsconfig-notification-alert enhancesnapshot
```
Comment (SonalOjha) [inline Makefile]: lib should be packaged with the particle, specially if the particle is deploying a lambda. Does removing this still package lib folder?

Comment (RajkumarShivage) [reply Makefile]: no, it will not package lib folder. if you want to package it, include it in mapping file.

Comment (SonalOjha) [reply Makefile]: The changes seems to only deploy managed-vpc and managed-os particles, none of which requires lib folder. This should be good.

Comment (bhargavapratik) [reply Makefile]: In Managed OS we deploy GetAMIId Lambda which requires lib folder.

Comment (RajkumarShivage) [reply Makefile]: No, it doesn't need lib folder.

**Comment by RajkumarShivage** [comment]
for datadog, only managed os is required. changes required for other particles, will do later(after datadog release)

**Comment by bhargavapratik** [comment]
LGTM

**Pull request merged**